requirements/system-requirements.md

```diff
@@ -1,22 +1,22 @@
 ---
 title: System Requirements
 ---
-# System Requirements
+This article lists the minimum hardware and software requirements for the installation of the SPDocKit. Depending on the scenario if you wish to use SPDocKit for SharePoint On-Premises or SharePoint Online view the corresponding section below.
 
-This article lists the minimum hardware and software requirements for the installation of the SPDocKit.
+## SharePoint On-Premises
 
 ### Requirements
 
 * The product needs to be installed (or started) on a **SharePoint 2016, SharePoint 2013 or SharePoint 2010 Server**
    * For SharePoint 2013 & 2010: SharePoint Foundation, Standard and Enterprise are supported.
    * You can install the product on WFE, Application server, Index or any other server in the farm.
```

Comment:
* You can install the product on a WFE, Application, Index or any other server in the farm.